src/router/routes.js

```diff
@@ -34,6 +34,7 @@ const routes = [
     children: [
       { path: '', component: () => import('pages/LibrariesPage.vue') },
       { path: '/libraries/:id', component: () => import('pages/LibraryPage.vue') },
+      { path: '/add-library', component: () => import('pages/AddLibraryPage.vue') },
```

Comment:
path: '/libraries/create'
OR
path: '/libraries/add'

Reply:
Can't do any of your proposals because
`'/libraries/:id'` -> id is a string of charactere so it will be no good if we have same page for two different functionnalities.